src/SndFileAudioFileReader.cpp

```diff
@@ -190,3 +190,57 @@ bool SndFileAudioFileReader::run(AudioProcessor& processor)
 }
 
 //------------------------------------------------------------------------------
+
+bool SndFileAudioFileReader::configure(int channels, int samplerate, const std::string& format)
+{
+    info_.seekable = 0;
+    info_.frames = 0;
+    info_.sections = 0;
+    
+    info_.channels = channels;
+    info_.format = SF_FORMAT_RAW;
+    info_.samplerate = samplerate;
+
+    if (format == "S8") {
```

Comment:
I'd prefer to use lower-case for these values, so they're the same as ffmpeg.

Reply:
okay.